extensions/wikia/AdEngine/js/provider/gpt/adDetect.js

```diff
@@ -170,7 +171,11 @@ define('ext.wikia.adEngine.provider.gpt.adDetect', [
 			expectAsyncHopWithSlotName = false,
 			expectAsyncSuccessWithSlotName = false,
 			expectAsyncSuccess = false,
-			successTimer;
+			successTimer,
+
+			slotFirstChildData,
+			pageParams,
+			slotParams;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the declaration of the unused 'slotParams' variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)